source/administration/command-line-tools.rst

```diff
@@ -1340,6 +1340,11 @@ mattermost sampledata
           -b, --bulk string                    Optional. Path to write a JSONL bulk file instead of loading into the database.
           -w, --workers int                    How many workers to run during the import. (default 2)
 
+  User Passwords
```

Comment:
Propose adding this to command description, and updating description to

```
Generate sample data and populate the Mattermost database. Supported in Mattermost v4.7 and later.

The command generates one user as the System Administrator with a username ``sysadmin`` and password ``Sys@dmin-sample1``. Other users are generated following an index, e.g. with username ``user-1`` and password ``SampleUs@r-%1``.
```

Reply:
^how is this? I wasn't sure if my description of the index is correct

@cpanato

Reply:
done, sounds good to me thanks!

Reply:
@cpanato Sorry that I wasn't clear, but I intended it to be in the command description, below the title

![image](https://user-images.githubusercontent.com/13119842/62468344-d255e280-b763-11e9-8633-2bfd2e579ca5.png)

Reply:
fixed @jasonblais